### PR TITLE
chore(ci): add comment to skip check and test and build for Draft PRs because often times they’re unfinished

### PR DESCRIPTION
**Pull request by jenstroeger**

### CODE REVIEW

File: .github/workflows/_build.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ env:
 
 jobs:
   build:
+    if: github.event.pull_request.draft == false
```
Comment (behnazh) [inline .github/workflows/_build.yaml:42]: I have found it very useful to build on draft PRs. So, I'm not sure I agree with this change.

**Review by behnazh — CHANGES_REQUESTED** [review]

**Comment by behnazh** [comment]
I'm not sure if this is a `fix` PR? :thinking:

File: .github/workflows/_build.yaml (hunk 1 above)
Comment (jenstroeger) [reply .github/workflows/_build.yaml:42]: In larger organizations where multiple people work on the same branch it’s common practice to _intentionally_ push changes that don’t pass checks & tests for safekeeping or to share current changes.

Furthermore, for private repositories where organizations pay for Github minutes, running checks & tests on changes that are known to fail can add up to significant amounts of money.

The purpose of a “draft” PR is to signal work in progress or, as Github [puts it](https://github.blog/2019-02-14-introducing-draft-pull-requests/), “code isn’t in any state to be judged”. Running checks and tests on unfinished work in progress doesn’t seem useful.

**Comment by jenstroeger** [comment]
> I'm not sure if this is a `fix` PR? 🤔

Are you suggesting to turn the `feat(ci)` into a `fix(ci)` here? What problem would it fix, though?

**Comment by behnazh** [comment]
> > I'm not sure if this is a `fix` PR? 🤔
> 
> Are you suggesting to turn the `feat(ci)` into a `fix(ci)` here? What problem would it fix, though?

I meant `feat`, not `fix`.

File: .github/workflows/_build.yaml (hunk 1 above)
Comment (behnazh) [reply .github/workflows/_build.yaml:42]: I would argue that such organizations should adjust the workflows based on their requirements.

>The purpose of a “draft” PR is to signal work in progress or, as Github [puts it](https://github.blog/2019-02-14-introducing-draft-pull-requests/), “code isn’t in any state to be judged”. Running checks and tests on unfinished work in progress doesn’t seem useful.

I disagree with this. Running CI on drat PRs can be very useful. Generally, we don't need to wait until the end to run CI tests.

Comment (jenstroeger) [reply .github/workflows/_build.yaml]: Commented out.

**Review by behnazh — APPROVED** [review]

**Pull request merged**